tests/test_database_update.py

```diff
@@ -12,53 +13,125 @@
 @pytest.mark.integration
 def test_database_update_s3():
     """
-    Tests custom database update for Kraken 2 and the standard database
+    Tests custom database update for bowtie 2 using S3 files
     """
-    output = toolchest.update_database(
-        database_path="s3://toolchest-integration-tests/arbitrary_directory/",
-        tool=toolchest.tools.Kraken2,
-        database_name="standard",
+    test_dir = "temp_test_db_update_s3_bowtie2"
+    os.makedirs(f"./{test_dir}", exist_ok=True)
+    output_dir_path = f"./{test_dir}"
+    output_file_path = f"{output_dir_path}/bowtie2_output.sam"
+
+    # Update DB
+    update_db_output = toolchest.update_database(
+        database_path=[
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.1.bt2",
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.2.bt2",
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.3.bt2",
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.4.bt2",
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.rev.1.bt2",
+            "s3://toolchest-integration-tests/databases/bowtie2-fruitfly-Dmel/Dmel_A4_1.0.rev.2.bt2",
+        ],
+        tool=toolchest.tools.Bowtie2,
+        database_name="integration_test_bowtie2_fruitfly",
+        database_primary_name="Dmel_A4_1.0",
     )
+    assert update_db_output.database_name == "integration_test_bowtie2_fruitfly"
+    assert update_db_output.database_version
 
-    assert output.database_name == "standard"
-    assert output.database_version
+    # Run job on updated DB
+    toolchest.bowtie2(
+        inputs="s3://toolchest-integration-tests/bowtie2/fruitfly-ncbi/GSM868349.fastq.gz",
+        output_path=output_dir_path,
+        database_name=update_db_output.database_name,
+        database_version=update_db_output.database_version,
+    )
+    print(os.path.getsize(output_file_path))  # replace w/ file size assertion
```

Comment:
Should this be a file size assertion?

Reply:
It should be, this is just a placeholder. I'll add actual values once we run the tests.